### PR TITLE
Changed usage check to use logs fetcher and get limit from var settings

### PR DESCRIPTION
**Pull request by AviadPepperi**
Changed usage check to:
1. Use logs fetcher.
2. Get daily usage limit from var settings.
3. Code formatting.

Change Monitor values to not be saved and read from meta_data.

### CODE REVIEW

File: server-side/installation.ts [hunk 1]
```diff
@@ -255,17 +256,19 @@ exports.upgrade = async (client: Client, request: Request) => {
...
             };
-            const responseSettingsTable = await service.papiClient.post('/addons/data/schemes', bodyADAL, headersADAL);
-            const distributor = await GetDistributor(service.papiClient);
-            const monitorLevel = await service.papiClient.get('/meta_data/flags/MonitorLevel');
-            const memoryUsageLimit = await service.papiClient.get('/meta_data/flags/MemoryUsageLimit');
+            const currentMemoryUsageLimit = (await monitorSettingsService.getMonitorSettings()).MemoryUsageLimit
+
+            const responseSettingsTable = await monitorSettingsService.papiClient.post('/addons/data/schemes', bodyADAL, headersADAL);
+            const distributor = await GetDistributor(monitorSettingsService.papiClient);
+            const monitorLevel = await monitorSettingsService.papiClient.get('/meta_data/flags/MonitorLevel');
```
Comment (nofarpepperi) [inline server-side/installation.ts:263]: arent you extracting the monitor level also for var settings?

File: server-side/addon-usage.ts [hunk 2]
```diff
@@ -39,43 +43,87 @@ export async function daily_addon_usage(client: Client, request: Request) {
...
     console.log('HealthMonitorAddon ended daily addon usage');
     return { DailyPassedLimit: dailyReport["PassedLimit"], MonthlyPassedLimit: monthlyReport["PassedLimit"] };
 }
 
-async function getcloudWatchLogs(service, now) {
-    const startTime = new Date(now - 24 * 3600 * 1000).setHours(0, 0, 0, 0);
-    const endTime = new Date(now).setHours(0, 0, 0, 0);
+async function getCloudWatchLogs(service, now, distributor) {
+
+    const startTime = new Date(new Date(now - TWO_DAYS_IN_MS).setUTCHours(0, 0, 0, 0)).toISOString();
```
[the remark this answers is not in the document]
Comment (AviadPepperi) [reply server-side/addon-usage.ts:64]: Added comment

File: server-side/addon-usage.ts [hunk 3]
```diff
@@ -39,43 +43,87 @@ export async function daily_addon_usage(client: Client, request: Request) {
...
+    const endTime = new Date(new Date(now - ONE_DAY_IN_MS).setUTCHours(0, 0, 0, 0)).toISOString();
+    const filter = 'Duration > 0' + ' and ' + `DistributorUUID='${distributor.UUID}'`;
+    const stats = "count(*) as Count, sum(Duration) as TotalDuration, sum(Duration)*LambdaMemorySize as MemoryUsage by AddonUUID,LambdaMemorySize";
+    const limit = 10000;
+
+    const statsForSyncAddons = {
```
[the remark this answers is not in the document]
Comment (AviadPepperi) [reply server-side/addon-usage.ts:70]: Extracted to a new function

File: server-side/installation.ts [hunk 4]
```diff
@@ -255,17 +256,19 @@ exports.upgrade = async (client: Client, request: Request) => {
...
+            const memoryUsageLimit = await monitorSettingsService.papiClient.get('/meta_data/flags/MemoryUsageLimit');
```
[the remark this answers is not in the document]
Comment (AviadPepperi) [reply server-side/installation.ts:264]: Deleted, was not used.

File: server-side/installation.ts (hunk 1 above)
Comment (AviadPepperi) [reply server-side/installation.ts:263]: Did you speak with him?

File: server-side/installation.ts [hunk 5]
```diff
@@ -7,14 +7,16 @@ If the result of your code is 'false' then return:
...
 import { Utils } from './utils.service'
 import { Client, Request } from '@pepperi-addons/debug-server'
 import jwtDecode from "jwt-decode";
 import MonitorSettingsService from './monitor-settings.service';
-import VarRelationService from "./relations.var.service";
+import VarRelationService, { VALID_MONITOR_LEVEL_VALUES } from "./relations.var.service";
+import Semver from "semver";
 
 const DEFAULT_MEMORY_USAGE = 5000000
+export const DEFAULT_MONITOR_LEVEL = 15
```
Comment (nofarpepperi) [inline server-side/installation.ts:19]: add comment what 15 represent

Comment (AviadPepperi) [reply server-side/installation.ts:19]: Added

**Review by nofarpepperi — APPROVED** [review]

**Pull request merged**